### PR TITLE
updating Git guidance as Databricks interface has changed

### PR DESCRIPTION
**Pull request by jen-machin**
## Overview of changes
- Updated guidance and screenshots
- Tidied up line breaks and weird newline escapes
- Modified all instances of "git" to be "Git" as per technical style guide 

## Why are these changes being made?
Databricks interface has changed. Repos are now called Git Folders and should be created in a different location to before. 

## Issue ticket number/s and link
Issue #95 

## Checklist before requesting a review
- [x] I have checked the contributing guidelines
- [x] I have checked for and linked any relevant issues that this may resolve
- [x] I have checked that these changes build locally
- [x] I understand that if merged into main, these changes will be publicly available

### CODE REVIEW

**Comment by jen-machin** [comment]
[Analysts Guide: updating Git in Databricks guidance due to interface changes](https://trello.com/c/XszU5UqX/1759-analysts-guide-updating-git-in-databricks-guidance-due-to-interface-changes)

File: ADA/git_databricks.qmd [hunk 1]
```diff
@@ -10,20 +10,18 @@ Guidance for analysts on how to connect Databricks to Github and Azure DevOps
```
Comment (cjrace) [inline ADA/git_databricks.qmd:10]: Line 7 in the subtext should be GitHub with a capital H

File: ADA/git_databricks.qmd [hunk 2]
```diff
@@ -52,29 +46,26 @@ The connection between git and Databricks is established using a secure access t
```
Comment (cjrace) [inline ADA/git_databricks.qmd:46]: Missed a git to Git capitalisation on line 45, there's two on there

File: ADA/git_databricks.qmd [hunk 3]
```diff
@@ -52,29 +46,26 @@ The connection between git and Databricks is established using a secure access t
...
+### Getting set up
 
+---
 
-### Access Tokens
+#### Access Tokens
 
+---
 
 Access tokens are long strings of numbers and letters that act like a password between two services. They identify the user and their permissions from one service to another.
```
Comment (cjrace) [inline ADA/git_databricks.qmd:71]: Line 86 (in the bullet points) there's a lower case git used

File: ADA/git_databricks.qmd [hunk 4]
```diff
@@ -127,28 +120,34 @@ You can then click Save at the bottom of the page, and now your connection betwe
...
 
-In the blue menu on the left, click Workspace, and then in the Workspace menu that appears, click Repos. The menus should look like this:
+Click the blue "Create" button in the top right corner, and then click "Git folder":
 
-![](../images/databricks-workspace-menu.png)
+![](../images/add-git-folder.png)
 
-On the Repos screen, click the grey "Add repo" button in the top right corner. The "Add Repo" window will appear:
 
-![](../images/databricks-add-repo.png)
```
Comment (cjrace) [inline ADA/git_databricks.qmd:139]: Should we be deleting the old files and reusing the name where possible? Would save us keeping all the old images in the latest repo too, they'll always be there in the Git history

**Review by cjrace — CHANGES_REQUESTED** [review]

File: ADA/git_databricks.qmd (hunk 2 above)
Comment (jen-machin) [reply ADA/git_databricks.qmd:46]: Oh my local version has this correctly, I wonder if I didn't commit it somehow? 
image

File: ADA/git_databricks.qmd (hunk 3 above)
Comment (jen-machin) [reply ADA/git_databricks.qmd:71]: Yeah this one is ok too for me locally - will commit and push again

File: ADA/git_databricks.qmd (hunk 4 above)
Comment (jen-machin) [reply ADA/git_databricks.qmd:139]: Yeah I agree with this - will delete unused ones now and will reuse names going forward

File: images/databricks-add-repo-new.png
Comment (cjrace) [inline images/databricks-add-repo-new.png:1]: Would it be better to rename this to `images/databricks-add-repo.png`?

File: images/databricks-main-repo-menu-new.png
Comment (cjrace) [inline images/databricks-main-repo-menu-new.png:1]: Similar here - should we take the 'new' off the name as that won't age well I don't think

**Review by cjrace — CHANGES_REQUESTED** [review]

File: images/databricks-add-repo-new.png
Comment (jen-machin) [reply images/databricks-add-repo-new.png:1]: Happy to do that now I've deleted the old one 😆 I didn't want to get myself confused 

image

File: images/databricks-main-repo-menu-new.png
Comment (jen-machin) [reply images/databricks-main-repo-menu-new.png:1]: Done!

**Review by cjrace — APPROVED** [review]

**Pull request merged**